users/spidey3/config.h

```diff
@@ -0,0 +1,11 @@
+#ifndef USERSPACE_CONFIG_H
+#define USERSPACE_CONFIG_H
+
+// Disable action_get_macro , since we don't use these
+// and it saves on space in the firmware.
+#define NO_ACTION_MACRO
```

Comment:
Just a heads up, a LOT of boards now have this by default.  You may not need to define it.

Also, if you have LTO enabled, you may not need it.

Reply:
Removed.